### ISSUE TITLE
[Emu] MQ2Nav not working

### ISSUE THREAD

**Issue by scotty0100** [macroquest/macroquest#644]
On the latest EMU build (v3.1.0.50) I am struggling to get /nav to work at all.

Next loads as it should with all the plugins seeming to work except for MQ2Nav when typing /nav it says "That is not a valid command. Please use /help."

![image](https://user-images.githubusercontent.com/4440226/194856931-b1ef1ba7-f223-4cb7-a783-d674efa4e5fb.png)

Tried unloading/reloading, fresh download with nothing custom in configs/macros. Am I missing something silly or is it not working for others also?

**Comment by brainiac** [comment]
This is #639.

Might be an issue with the built release. Does it work if built from source?

**Comment by scotty0100** [comment]
Nope wasn't that. I resolved it by moving the EQ ROF2 Install from Program Files to just the C:\ drive... and then it worked.

Pretty confused as to why that needed to be done but it works now.

**Comment by brainiac** [comment]
sounds like you might've been accidentally running an older version.

**Comment by scotty0100** [comment]
I got this again today. I did a lot of testing and found out it was my eqclient.ini, I went through a lot of settings and it was this one causing the problem:

WindowedMode=TRUE

If this is set to FALSE, I can boot up EQ and the Lua blue autologin window will not load and upon loading into game /nav won't work. If it is set to TRUE.. everything works as normal.

Is this just me? Curious if that setting breaks for others.
